rhobs/alerting/data_plane/prometheus.exporter_alerts.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: konflux-exporter-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: exporter_alerts
+    interval: 5m
+    rules:
+    - alert: KonfluxUpAlert
+      expr: absent_over_time(konflux_up{namespace="appstudio-grafana-datasource-exporter"}[5m]) == 1
+      for: 5m
+      labels:
+        severity: warning
+      annotations:
+        summary: Exporter metric 'konflux_up' is missing.
+        description: >-
+          The Prometheus exporter metric 'konflux_up' for namespace
+          "appstudio-grafana-datasource-exporter" on cluster
+          {{ $labels.source_cluster }} is missing for more than 30 minutes.
```

Comment:
It was 5 minutes window for 5 minutes, I think it should not mean 30 minutes. Using `absent` should make it simpler to understand the time window.